secator/query/json.py

```
@@ -94,26 +103,23 @@ def _get_workspace_path(self) -> Path:
 		workspace_folder = sanitize_folder_name(self.workspace_id)
 		return self.reports_dir / workspace_folder
 
-	def _load_all_findings(self) -> List[Dict[str, Any]]:
-		"""Load all findings from workspace JSON files, or return pre-loaded/cached results."""
+	def _load_all_findings(self) -> Generator[Dict[str, Any], None, None]:
+		"""Yield findings from workspace JSON files, or iterate pre-loaded results."""
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)